<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroAppClass</class>
 <widget class="QMainWindow" name="MacroAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Breakfast Macros</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MacroApp.qrc">
    <normaloff>:/MacroApp/logo.png</normaloff>:/MacroApp/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="MacroApp.qrc">
      <normaloff>:/MacroApp/iconwhite.png</normaloff>:/MacroApp/iconwhite.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="newMacroButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>New Macro...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteMacroButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Macro</string>
    </property>
   </widget>
   <widget class="QLabel" name="bpart1">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Breakfast Macros</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <item>
     <property name="text">
      <string>d</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MacroApp.qrc"/>
 </resources>
 <connections/>
</ui>
